data-processor/src/test/groovy/io/micronaut/data/processor/sql/BuildUpdateSpec.groovy

```diff
@@ -524,7 +524,7 @@ interface BookRepository extends GenericRepository<Book, Long> {
         when:
             def updateReturningCustomMethod = repository.findPossibleMethods("updateReturning").findFirst().get()
         then:
-            getQuery(updateReturningCustomMethod) == 'UPDATE "book" SET "author_id"=?,"genre_id"=?,"title"=?,"total_pages"=?,"publisher_id"=?,"last_updated"=? WHERE ("id" = ?) RETURNING "id","author_id","genre_id","title","total_pages","publisher_id","last_updated"'
```

Comment:
Returning also returns only fields belonging to the entity. This might be the most significant change, not sure if returning could have `@Join` annotation to tell Micronaut Data to return joined entities?